docs/docs/coverage/iac/terraform.md

```diff
@@ -8,18 +8,23 @@ Trivy supports the scanners listed in the table below.
 
 It supports the following formats:
 
-|  Format   | Supported |
-|:---------:|:---------:|
-|   JSON    |     ✓     |
-|    HCL    |     ✓     |
-| Plan JSON |     ✓     |
-
-Trivy can scan the results of `terraform plan`.
-You can scan by passing the file generated as shown below to Trivy:
-
+|     Format    | Supported |
+|:-------------:|:---------:|
+|     JSON      |     ✓     |
+|      HCL      |     ✓     |
+|   Plan JSON   |     ✓     |
+| Plan Snapshot |     ✓     |
```

Comment:
Is it easier if we simply parse the input within Trivy and decide if the user has passed in a Plan JSON or a Plan Snapshot?

 I feel having to know which kind of Terraform Plan scanner to use is a little overwhelming for the user as compared to just  "scanning the plan" and letting Trivy decide what the input is.

Another example would be: today we scan terraform, without the user telling us if it's HCL v1 or v2.

WDYT?

Reply:
Trivy automatically detects the type of configuration, so the user does not need to worry about selecting the type of scanner. This page describes the Trivy coverage.

Reply:
Sorry I misread this. It makes sense now.